kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/BaseGroup.java

```diff
@@ -412,6 +416,10 @@ protected AuthzGroup getAzg()
 
 	public void addMember(String userId, String roleId, boolean active, boolean provided)
 	{
+		if(this.isLocked()) {
+			M_log.error("Error, cannot add a member into a locked group");
```

Comment:
Can you put role and user into the error?

M_log.error("Error, cannot add user {} with role {} into a locked group", userId, roleId);